recipes/igblast-parser/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "igblast-parser" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+
+  url: "https://github.com/aerijman/{{ name }}/releases/download/v0.0.3/igblast_parser-{{ version }}.tar.gz"
+  sha256: c688be522cb9a887dca8baf4a07cc6805ccacda0d639a9743ba59c497c838f56
+
+build:
+  number: 0
+  noarch: python
+  script: $PYTHON setup.py install --single-version-externally-managed --record=record.txt 
+
+requirements:
```

Comment:
I think you will need to add, a build requirement to bring gcc in for you.

```
  build:
    - {{ compiler('cxx') }}
```

Reply:
Thank you very much for the help!

Why cxx?

Reply:
I didn't know you were changing the yaml. I kept yours. Just pushed to activate circleci. I thought that I might have mess with the check. I hope will be fine!

Reply:
I did not include the .c file to allow for users with or without cython. I will add module1.c file.

Reply:
mmmm... Just noticed that module1.c **IS** in there...

Reply:
are there any C++ files or just this C file?

Reply:
just the c

Reply:
In the tar.gz I don't see the `.c` file only the `.pyx`.
It's generated from cython.

Should I add manually the `.c` file in the `.tar.gz`?